python/mxnet/ndarray.py

```diff
@@ -1104,48 +1103,45 @@ def arange(start, stop=None, step=1.0, repeat=1, ctx=None, dtype=mx_real_t):
     """Returns evenly spaced values within a given interval.
 
     Values are generated within the half-open interval [start, stop). In other
-    words, the interval includes start but excludes stop. For integer
-    arguments, the function is equivalent to the built-in Python function ``range``
-    and to ``numpy.arange``, but returns an ``NDArray``.
+    words, the interval includes start but excludes stop. The function is
+    similar to the built-in Python function ``range`` and to ``numpy.arange``,
+    but returns an ``NDArray``.
 
     Parameters
     ----------
-    start : int, optional
-        An optional start of interval, the default value is 0.
-    stop : int
-        The end of interval.
-    step : int, optional
-        A optional spacing between values, the default value is 1.
+    start : float, optional
```

Comment:
@mli, @piiswrong 
Is it possible to extract the documentation from the internal C++ method ? Seeing that the type was int instead of float, I think Its easy to  misinterpret the datatypes when you have write up documentation  for other language binding.